www/md_base/src/styles/about.less

```diff
@@ -0,0 +1,35 @@
+.about{
+  padding: 20px;
```

Comment:
That is a lot of custom styles for one page.

I'd prefer we stick with a common style